File: launchdarkly/ruby/src/GetRepositoriesExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::CodeReferencesApi.new.get_repositories

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling CodeReferencesApi#get_repositories: #{e}"
end

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetRepositoriesExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetRepositoriesExample
{
    fun getRepositories()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        try
        {
            val response = CodeReferencesApi().getRepositories();

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling CodeReferencesApi#getRepositories")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling CodeReferencesApi#getRepositories")
            e.printStackTrace()
        }
    }
}
